Incremental.c

```diff
@@ -1766,6 +1772,6 @@ int IncrementalRemove(char *devname, char *id_path, int verbose)
 			    verbose, 0, UOPT_UNDEFINED, 0);
 
 	close(mdfd);
-	free_mdstat(ent);
+	free_mdstat(mdstat);
```

Comment:
Shouldn't mdstat be freed with  each return?

Reply:
Yes it should